electrum_nmc/electrum/gui/qt/main_window.py

```diff
@@ -3360,9 +3363,24 @@ def update_buy_names_preview(self):
         identifier = self.buy_names_new_name_lineedit.text().encode('ascii')
```

Comment:
I think we can use the hex textbox instead here?

Reply:
I am presently updating the hex text box using this line. 
Is it essential to set the hex text box and then retrieve its content again to update the name preview?

Reply:
Alternatively, I can implement a separate function to read the ASCII text_box (For the Hex text_box)